apps/settings/js/startup.js

```diff
@@ -29,7 +29,8 @@
 
     if (this._settings) {
       this._init();
-      if (typeof customPanelHandlerFunc !== 'undefined') {
+      if (customPanelHandlerFunc &&
+          typeof customPanelHandlerFunc !== 'undefined') {
```

Comment:
The second part is now redundant, I believe, if customPanelHandlerFunc is truthy, it will never have a type of undefined.